src/6_branch.js

```diff
@@ -2024,4 +2051,41 @@ Branch.prototype['trackCommerceEvent'] = wrap(callback_params.CALLBACK_ERR, func
 	done();
 });
 
-
+/**
+ * @function Branch.disableTracking
+ * @param {Boolean} disableTracking - _required_ - true or false depending on whether the user would like to remain private or not
```

Comment:
We should explicitly state that passing in true will disable tracking and passing false will enable tracking and that it will overwrite the options passed into init

Reply:
Also looks like below they have they option to pass in no arguments and it would be the same as passing in true. We should spell that out here if that's the case